aosp_diff/preliminary/system/core/0001-adb-dbc-tty-add-adb-over-dbc-tty-support.patch

```diff
@@ -0,0 +1,491 @@
+From c733fe2c43e610392dfa1c28ef07fab72d335c78 Mon Sep 17 00:00:00 2001
+From: Prabhat Chand Pandey <prabhat.chand.pandey@intel.com>
+Date: Sun, 22 Nov 2020 12:59:30 +0530
+Subject: [PATCH] adb: dbc: tty: add adb over dbc tty support
+
+Added adb over dbc tty support in adb framework.
+This implementation is based on Blocking I/O calls.
+Set "setprop persist.vendor.sys.usb.adbover" android
+property to "dbc" to use adb over dbc tty otherwise
+set this property to "dwc" (which is the default value
+of this property) to use adb over normal USB (which need
+USB Device Mode IP to be enabled).
```

Comment:
Commit message not inline with the changes done as part of system/core changes

Reply:
ACK